Library/Formula/cookiecutter.rb

```diff
@@ -12,14 +11,44 @@ class Cookiecutter < Formula
     sha1 "4c69f62dfffca5508feb0a5753dd0d4b7efb8ebe" => :mountain_lion
   end
 
-  depends_on :python if MacOS.version <= :snow_leopard
+  depends_on :python
```

Comment:
Don't change this, it only needs this for Python 2.7 on versions earlier than Lion.